docs/events/wg_workshops/2023-06-28-june-meeting/index.md

```diff
@@ -0,0 +1,132 @@
+| 16:40 - 16:45 | Breakout discussions 2 shareout      | Hari Sood (Turing)                                                                             |                                                                                                                        |
+| 16:45 - 17:00 | Wrap up & Working Group breakouts    | Hari Sood (Turing)                                                                             |                                                                                                                        |
+
+## :wave: Attendees
+
+_Please add your name below (if you want to), your institution, your email (E), github username (GH), twitter (T) or any other info (if you're happy to share with the group) and your favourite thing about Summer, in the form of `Name / Institution / (E) (GH) (T) / Favourite thing about Summer`_
+
+_These notes will be made public after the event (with identifiers like email, GitHub and Twitter handles removed). If you'd prefer for your details to be removed from the public version, please put :shushing_face: after your name below!_
+
+- James Robinson, The Alan Turing Institute
+- Sarah Waters, Security Governance Risk and Compliance Officer, University of Oxford
+- David Sarmiento, The Alan Turing Institute
+- Hari Sood, Alan Turing Institute
+- Simon Li, Health Informatics Centre, University of Dundee
+- Stuart Wheater, TRE-FX (Arjuna Technologies)
+- Sophie McCall, Research Data Scotland
+- Balint Stewart, DARE UK
+- Fatemeh Torabi, DPUK- SAIL Wales Swansea University
+- Will Crocombe, RISG consulting, IG working group
+- Ruairidh MacLeod, EPCC (University of Edinburgh)
+- Bianca Prodan, EPCC (University of Edinburgh)
+- Seb Bacon, Bennett Institute for Applied Data Science, University of Oxford
+- Ed Chalstrey, The Alan Turing Institute, Research Engineering Group & Data Safe Haven Project
+- Arron Lacey, The Alan Turing Institute
+- Amy Dillon, BRC, University of Bristol
+- Jackie Caldwell, eDRIS Public Health Scotland
+- Jonathan Flavell, ONS Secure Research Service
+- Lucy Cheesman, University of Sheffield
+- Jamila Lombardo-Dearnaley, ONS Secure Research Service
+- Tim Hubbard, KCL/GenomicsEngland/HDRUK
+- Jim Madge, Alan Turing Institute
+- Aida Sanchez, UCL, Centre for Longitudinal Studies (CLS)
+- Adam Huffman, Big Data Institute, University of Oxford
+- Fergus McDonald, DARE UK
+- Rosie Wakeham, HDR UK
+- Simone Croft, University of Sheffield
+- Antony Chuter, Public member, University of Dundee, SATRE
+- Rob Baxter, DARE UK
+- Matt Craddock, Alan Turing Institute
+- Donald Scobbie, EPCC (University of Edinburgh)
+- Michelle Amugi, DARE UK
+- James Sutherland, Health Informatics Centre, University of Dundee
```

Comment:
Do you think we should remove the attendees list?

Reply:
That's my preference, but I left it in because the hackmd says
> *These notes will be made public after the event (with identifiers like email, GitHub and Twitter handles removed). If you'd prefer for your details to be removed from the public version, please put :shushing_face: after your name below!*

Should I take it out?